it/sender_util.go

```diff
@@ -2,246 +2,106 @@ package it
 	}
+	return result, nil
+}
 
-	defer resp.Body.Close()
-
-	body, err := io.ReadAll(resp.Body)
-
-	if err != nil {
-		fmt.Println("Body read err")
-		fmt.Printf("%s", err)
+func makeUrlValues(beaconDataMap map[string]string) url.Values {
+	result := url.Values{}
+	for k, v := range beaconDataMap {
+		result.Set(mapKey(k), v)
 	}
-
-	fmt.Println(string(body))
+	return result
 }
 
-func getRealBeaconsOldStyle(path string) []url.Values {
-	files, _ := filepath.Glob(path)
+func mapKey(value string) string {
+	keyPrefix := extractBeaconPrefix(value)
 
-	valuesSlc := []url.Values{}
-
-	for i, s := range files {
-		fmt.Println(i, s)
-
-		fContent, err := os.ReadFile(s)
-		if err != nil {
-			log.Fatalf("unable to read file: %v", err)
-		}
-
-		var fJson interface{}
-		unmrErr := json.Unmarshal(fContent, &fJson)
-
-		if unmrErr != nil {
-
-			fmt.Println("Bad file")
-			fmt.Println(unmrErr)
-		}
-
-		for _, value := range fJson.([]interface{}) {
-
-			bMap := value.(map[string]interface{})
-
-			var b interface{}
-			unmrErr2 := json.Unmarshal([]byte(bMap["beacon_data"].(string)), &b)
-
-			if unmrErr2 != nil {
-				log.Printf("Bad beacon_data in file: %v", unmrErr2)
-				continue
-			}
-
-			beaconData := b.(map[string]interface{})
-
-			reqD := make(url.Values)
-
-			for bK, bV := range beaconData {
-				bKey := bK
-
-				keyPart := ""
-
-				if len(bKey) > 3 {
-					keyPart = bKey[0:3]
-				}
-
-				if keyPart == "nt_" || bKey == "created_at" || bKey == "t_resp" || bKey == "t_done" || bKey == "t_page" || bKey == "t_other" {
-					reqD.Set(bK, bV.(string))
-					continue
-				}
-
-				reqD.Set(strings.ReplaceAll(bKey, "_", "."), bV.(string))
-			}
-
-			valuesSlc = append(valuesSlc, reqD)
-		}
+	if keyPrefix == "nt_" {
+		return value
 	}
 
-	return valuesSlc
+	switch value {
+	case "created_at":
+		return value
+	case "t_resp":
+		return value
+	case "t_done":
+		return value
+	case "t_page":
+		return value
+	case "t_other":
+		return value
+	default:
+		return strings.ReplaceAll(value, "_", ".")
+	}
 }
 
-func getRealBeaconsNewStyle(path string) []url.Values {
-	files, _ := filepath.Glob(path)
-
-	valuesSlc := []url.Values{}
-
-	for i, s := range files {
-		fmt.Println(i, s)
-
-		file, err := os.Open(s)
-		if err != nil {
-			log.Fatalf("unable to read file: %v", err)
-		}
-
-		defer file.Close()
-
-		scanner := bufio.NewScanner(file)
-
-		for scanner.Scan() {
-
-			var b interface{}
-
-			unmrErr := json.Unmarshal([]byte(scanner.Text()), &b)
-
-			if unmrErr != nil {
-				log.Printf("Bad beacon_data in file: %v", unmrErr)
-				continue
-			}
-
-			beaconData := b.(map[string]interface{})
-
-			reqD := make(url.Values)
-
-			for bK, bV := range beaconData {
-				// Umarshal later request_headers
-				reqD.Set(bK, bV.(string))
-			}
-
-			valuesSlc = append(valuesSlc, reqD)
-		}
-
-		if err := scanner.Err(); err != nil {
-			log.Fatal(err)
-		}
+func extractBeaconPrefix(value string) string {
```

Comment:
used for oldStyleBeaconSender. move it to file old_style_beacon_sender.go and make it part of struct oldStyleBeaconSender